integration/component/pre-suite/05_install_dev_r10k.rb

```diff
@@ -1,10 +1,12 @@
 test_name "Install PE r10k" do
 
   step "Install PE r10k" do
+    variant, version, arch, codename = master['platform'].to_array
+    if variant == 'ubuntu' && version.split('.').first.to_i >= 18
+      on master, "echo 'Acquire::AllowInsecureRepositories \"true\";' > /etc/apt/apt.conf.d/90insecure"
```

Comment:
Oh, nit: we're not installing the signing key with this commit, we're instructing apt to ignore the fact we don't have it.